### PR TITLE
Bugfix/pelagos 3431 remove foreign key constraint

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php [hunk 1]
```diff
@@ -380,7 +380,7 @@ private function createNewDatasetSubmission(DatasetSubmission $datasetSubmission
         $datasetSubmission = new DatasetSubmission($datasetSubmission);
         $reviewedBy = $this->getUser()->getPerson();
         $reviewStartTimeStamp = new \DateTime('now', new \DateTimeZone('UTC'));
-        $datasetSubmissionReview = new DatasetSubmissionReview($datasetSubmission, $reviewedBy, $reviewStartTimeStamp);
+        $datasetSubmissionReview = new DatasetSubmissionReview($reviewedBy, $reviewStartTimeStamp);
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php]: I thought there would be more places, but this is the only one.

File: app/DoctrineMigrations/Version20181219204218.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE dataset_submission_review DROP CONSTRAINT fk_3fa5c62f8488ba54');
+        $this->addSql('DROP INDEX uniq_3fa5c62f8488ba54');
+        $this->addSql('ALTER TABLE dataset_submission_review DROP dataset_submission_id');
```
Comment (fightingtexasaggie) [inline app/DoctrineMigrations/Version20181219204218.php]: Considering adding SQL to populate the new field before loosing reference to the old one.

Comment (praneethpr) [reply app/DoctrineMigrations/Version20181219204218.php]: I've added that update statement to the migration, thanks mike

File: src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php (hunk 1 above)
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php]: Need to set DatasetSubmissionReview object to DatasetSubmission, and it needs to updated after the DatasetSubmissionReview is created

File: src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php [hunk 3]
```diff
@@ -379,8 +379,6 @@ private function createNewDatasetSubmission(DatasetSubmission $datasetSubmission
         // The latest submission is complete, so create new one based on it.
         $datasetSubmission = new DatasetSubmission($datasetSubmission);
         $reviewedBy = $this->getUser()->getPerson();
-        $reviewStartTimeStamp = new \DateTime('now', new \DateTimeZone('UTC'));
-        $datasetSubmissionReview = new DatasetSubmissionReview($datasetSubmission, $reviewedBy, $reviewStartTimeStamp);
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/DatasetReviewController.php]: Could have left the datasetSubmission in, and then called the setting of the review on it with this.

**Comment by fightingtexasaggie** [comment]
This is now working.

**Pull request merged**